agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/configuration/Configuration.java

```diff
@@ -122,17 +122,23 @@
         public List<ProcessorAction> actions; // specific for processor type "attributes"
         public NameConfig name; // specific for processor types "log" and "span"
 
-        private static void isValidRegex(String value) {
+        private static void isValidRegex(String value) throws FriendlyException {
             try {
                 Pattern.compile(value);
             } catch (PatternSyntaxException exception) {
-                throw new ConfigurationException("User provided processor config do not have valid regex:" + value);
+                throw new FriendlyException("ApplicationInsights Java Agent failed to start.",
+                                "User provided processor configuration do not have valid regex:" + value,
```

Comment:
I don't think "User provided" is needed, how about: (and applies below too)
```suggestion
                                "Telemetry processor configuration does not have valid regex:" + value,
```